### ISSUE TITLE
\setmsdata{\ihead} msdata Position in \ihead section of the KOMA class

### ISSUE THREAD

**Issue by Arcerius** [maieul/ledmac#750]
Version of reledmac 2.26.5

Problem:
In the scrbook I want to put the \msdata command of reledmac in the \ihead section of the KOMAclass. Setting \setmsdataposition{\ihead} doesn't work.  

MWE:

\documentclass{scrbook}
\usepackage{fontspec}
\usepackage[latin]{babel}
\usepackage[series={A,B}]{reledmac}
\setmsdataposition{\ihead}
\usepackage{scrlayer-scrpage}
\pagestyle{scrheadings}
\cehead{AUTHOR}
\cohead{TITLE}
\ohead{\pagemark}
\ihead{}
\pagenumbering{arabic}

\begin{document} 
\beginnumbering 
\pstart
\msdata{AB}sit transitorem ipsum dolor 
\edtext{amet}{\lemma{}\Bfootnote[nosep]{amat}}\stopmsdata{} sit transitorem ipsum dolor 
amet 
\pend
\endnumbering
\end{document}

**Comment by maieul** [comment]
of course, that is position related to the footnote. Not position relates to header/footer. I need to add new feature to get the ms data in header (with all class), but I have no time yet.
